From: Your Name <you@example.com>
Date: Wed, 21 Feb 2024 09:16:27 +0000
Subject: Temp disable network service windows

---
 sandbox/policy/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sandbox/policy/features.cc b/sandbox/policy/features.cc
--- a/sandbox/policy/features.cc
+++ b/sandbox/policy/features.cc
@@ -217,6 +217,6 @@ SET_CROMITE_FEATURE_ENABLED(kUseRendererProcessPolicy);
 SET_CROMITE_FEATURE_ENABLED(kRestrictRendererPoliciesInBaseline);
 #endif
 #if BUILDFLAG(IS_WIN)
-SET_CROMITE_FEATURE_ENABLED(kNetworkServiceSandbox);
+//SET_CROMITE_FEATURE_ENABLED(kNetworkServiceSandbox);
 #endif
 }  // namespace sandbox::policy::features
--
